browser/src/canvas/sections/CommentSection.ts

```diff
@@ -1422,15 +1422,46 @@ export class Comment extends CanvasSectionObject {
 	}
 
 	public onClick (point: cool.SimplePoint, e: MouseEvent): void {
-		if (app.map._docLayer._docType === 'presentation' || app.map._docLayer._docType === 'drawing') {
+		const docType = app.map._docLayer._docType;
+
+		if (['presentation', 'drawing'].includes(docType)) {
 			this.sectionProperties.commentListSection.selectById(this.sectionProperties.data.id);
 		}
-		else if (app.map._docLayer._docType === 'text') {
+		else if ('text' === docType) {
 			const mousePoint = point.clone();
 			mousePoint.pX += this.myTopLeft[0];
 			mousePoint.pY += this.myTopLeft[1];
 			app.activeDocument.mouseControl.onClick(mousePoint, e);
 		}
+		else if ('spreadsheet' === docType) {
+			/*
+				On Calc, comments are shown when user focuses to the cell or moves the mouse pointer over the cell.
+				Comment class (this class) covers the cell area. So the document is blocked now.
+				We need to pass the click event to the document.
+			*/
+			point.pX += this.position[0];
+			point.pY += this.position[1];
+
+			app.map._docLayer._postMouseEvent('buttondown', point.x, point.y, 1, 1, 0);
```

Comment:
This is acceptable, but do you know why is this necessary? I mean, if the comment doesn't actually cover the cell area, then why does this not work out of the box? Do we have some huge transparent frame around the comment that makes the comment get the click? Is this a bug in some hit testing code?